s3.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_s3_bucket" "core-data" {
+  bucket = "mhclg-core-data-upload"
+  acl    = "private"
```

Comment:
Perhaps want the data encrypted at rest as well, since we may be using real data.

Reply:
Good point